app/views/returns/NonExportedHumanMedicinesPlasticPackagingWeightView.scala.html

```diff
@@ -26,12 +26,12 @@
     formHelper: FormWithCSRF
 )
 
-@(amount: Long, form: Form[Long], mode: Mode, directlyExportedYesNoAnswer: Boolean)(implicit request: Request[_], messages: Messages)
+@(amount: Long, form: Form[Long], mode: Mode, directlyExportedYesNoAnswer: Boolean, exportedByAnotherBusinessAnswer: Boolean)(implicit request: Request[_], messages: Messages)
 
 @paragraphContent = { @paragraphBody(messages("nonExportedHumanMedicinesPlasticPackagingWeight.paragraph.1")) }
 
 @heading = @{
-    if(directlyExportedYesNoAnswer) {
+    if(directlyExportedYesNoAnswer || exportedByAnotherBusinessAnswer) {
```

Comment:
^^

Reply:
yep we could do that. maybe as part of another piece of work as I would like to get this in for the navigation. I will put a todo comment